addons/stock_account/models/product.py

```diff
@@ -14,6 +14,32 @@ class ProductTemplate(models.Model):
 
     cost_method = fields.Selection(related="categ_id.property_cost_method", readonly=True)
     valuation = fields.Selection(related="categ_id.property_valuation", readonly=True)
+    lot_valuated = fields.Boolean(
```

Comment:
I understand that it could be enable only on a subset of products but I would still allow the option on the category

Reply:
It's confusing to have a part of the configuration on different objects